### PR TITLE
[Magic WAN] Added tutorial for Aruba EdgeConnect

### PR DESCRIPTION
**Pull request by dcpena**
Added tutorial for Aruba EdgeConnect to address [PCX-3431](https://jira.cfops.it/browse/PCX-3431).

### CODE REVIEW

File: content/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints.md [hunk 1]
```diff
@@ -25,6 +25,25 @@ To configure the Anycast GRE or IPsec tunnels between Cloudflare and your locati
 
 For an example Anycast GRE or IPsec tunnel configuration, see [Anycast GRE configuration example](/magic-wan/reference/configuration-examples/#gre-tunnel-configuration-example).
 
+### Add GRE tunnels
+
+1.  Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2.  Next to **GRE tunnels and static routes configuration**, click **Configure**.
+3.  From **GRE tunnels**, click **Create**.
+4.  On the **Add GRE tunnels** page, fill out the information for your GRE tunnel.
+5.  _(Optional)_ We recommend you test your tunnel before officially adding it. To test the tunnel, click **Test tunnels.**
```
Comment (kodster28) [inline content/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints.md]: Should be consistent with the other optional in `assign-tunnel-priorities`

Comment (kodster28) [inline content/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints.md]:
```suggestion
5.  _(Optional)_ We recommend you test your tunnel before officially adding it. To test the tunnel, click **Test tunnels**.
```

File: content/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints.md [hunk 2]
```diff
@@ -25,6 +25,25 @@ To configure the Anycast GRE or IPsec tunnels between Cloudflare and your locati
...
+6.  To add multiple tunnels, click **Add GRE tunnel** for each new tunnel.
+7.  After adding your tunnel information, click **Add tunnels** to save your changes.
+
+### Edit GRE tunnels
+
+1.  From **GRE tunnels**, locate the GRE tunnel you want to modify and click **Edit**. To edit multiple tunnels, select the checkboxes for each tunnel and then click **Edit selected tunnels**.
+2.  On the **Edit GRE tunnels** page, fill out the fields you want to modify.
+3.  _(Optional)_ We recommend you test your tunnel before officially adding it. To test the tunnel, click **Test tunnels.**
```
Comment (kodster28) [inline content/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints.md]:
```suggestion
3.  _(Optional)_ We recommend you test your tunnel before officially adding it. To test the tunnel, click **Test tunnels**.
```

File: content/magic-wan/tutorials/aruba-edgeconnect.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+title: Aruba EdgeConnect SD-WAN
```
Comment (kodster28) [inline content/magic-wan/tutorials/aruba-edgeconnect.md:2]: Ugh, literally got the beach boys in my head now

File: content/magic-wan/tutorials/aruba-edgeconnect.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Below are examples of the east_branch deployment on the Orchestrator.
+
+![East branch deployment configuration](/magic-wan/static/east-branch-deployment.png)
+
+![Modify Passthrough tunnel configuration](/magic-wan/static/east-branch-passthrough.png)
+
+## Configure Anycast tunnels
+
+Refer to [Specify tunnel endpoints](/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints) for more information about configuring your Anycast tunnels.
```
Comment (kodster28) [inline content/magic-wan/tutorials/aruba-edgeconnect.md]:
```suggestion
Refer to [Specify tunnel endpoints](/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints/) for more information about configuring your Anycast tunnels.
```

**Review by kodster28 — APPROVED** [review]

File: content/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints.md (hunk 1 above)
Comment (dcpena) [reply content/magic-wan/get-started/configure-tunnels/specify-tunnel-endpoints.md]: Fixed

File: content/magic-wan/tutorials/aruba-edgeconnect.md (hunk 3 above)
Comment (dcpena) [reply content/magic-wan/tutorials/aruba-edgeconnect.md:2]: Hahahaha

**Pull request merged**